_docs/_latest/use/explain.md

```diff
@@ -0,0 +1,274 @@
+
+```bash
+$ forseti-client-XXXX-vm> forseti explainer check_policy <RESOURCE_NAME> <PERMISSION_NAME> <MEMEBER_NAME>
+```
+
+For example, the following query returns True or False to indicate if the member has permissions on the resource:
+
+```bash
+$ forseti-client-XXXX-vm> forseti explainer check_policy organizations/1234567890 iam.roles.get user/user1@gmail.com
+```
+
+#### List member access
+The following command lists all resources that can be accessed by a member by a direct binding:
+
+```bash
+$ forseti-client-XXXX-vm> forseti explainer access_by_member user/<USER_NAME>
+```
+
+By default, resource_expansion is not performed. For example, `user/foo` has `roles/owner` on the `organization/1234567890` and no other policies. The above command will only show `organization/1234567890`.
+
+To enable resource expansion pass in the argument `--expand_resource true`. Forseti will perform resource expansion and list all resources accessible by `user/<USER_NAME>` through a direct or indirect binding.
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_member user/<USER_NAME> --expand_resource true
+```
+ 
+With resource expansion enabled, the example above will return `organizations/1234567890` and all folders, projects, VMs, and other resources in it.
+
+To constrain the result to a certain type of permission, use the following command:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_member user/<USER_NAME> iam.roles.get
+```
+
+The above query will list all resources that can be accessed by the member with a binding of `iam.roles.get`.
+
+#### List resource permissions
+
+The following command lists all members that can access a resource by a direct binding:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME>
+```
+
+Similar to the `access_by_member` command, group expansion is not performed by default. For example, `group/bar` has `roles/owner` on the `organization/1234567890` and no other policies. Without group expansion, the following example will only return `group/bar`:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource organizations/1234567890
+```
+
+To enable group expansion pass in the `--expand_group` argument. Forseti will perform group expansion and list all members that can access the resource, even if the access is granted because a member is in a group.
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME> --expand_group true 
+```
+
+With group expansion enabled, the example above will list `group/bar` and `user/foo` if `user/foo` is a member in `group/bar`.
+
+To constrain the result to a certain type of permission, pass in the permission:
+
+```
+$ forseti-client-XXXX-vm> forseti explainer access_by_resource <RESOURCE_NAME> <PERMISSION> 
+```
+
+#### Access member or resource by permission
+
+You can specify a permission, such as `p`, and list `<members, resource, role>` pairs that have a relation.
```

Comment:
I'm not sure what's happening here. Can you provide some more detail, or should we sync to discuss?

Reply:
Updated the wording to make it easier to understand.